infoblox_client/feature.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2015 Infoblox Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import re
+import six
+
+from infoblox_client import exceptions as ib_ex
+
+FEATURE_VERSIONS = [
+    {'feature': 'create_ea_def', 'min_ver': '2.2'},
+    {'feature': 'cloud_api', 'min_ver': '2.0'},
+    {'feature': 'member_ipv6_setting', 'min_ver': '2.0'},
+    {'feature': 'member_licenses', 'min_ver': '2.0'}
+]
+
+
+class Feature(object):
+    def __init__(self, version, feature_versions=FEATURE_VERSIONS):
```

Comment:
you should not set default argument to mutable type because of [1].
It should be:

    def __init__(self, version, feature_versions=None):
        if feature_versions is None:
            feature_versions = FEATURE_VERSIONS

[1] https://pythonconquerstheuniverse.wordpress.com/2012/02/15/mutable-default-arguments/